docker/compose.yml

```diff
@@ -16,3 +16,12 @@ services:
       - "127.0.0.1:21325:21326" # macos needs proxy to override the "origin" of the trezord request
     environment:
       - DISPLAY=docker.for.mac.host.internal:0
+
+  trezor-user-env-regtest:
+    container_name: trezor-user-env-regtest
+    image: registry.gitlab.com/satoshilabs/trezor/trezor-user-env/trezor-user-env-regtest
+    ports:
+      - "19021:19021" # internal blockbook port
+      - "19121:19121" # public blockbook port for website dashboard
+      - "18021:18021" # bitcoin-d rpc port
```

Comment:
As per `bitcoin-cli` help, regtest has port 18443 as default. We should follow that.

Reply:
yes, but we are trying to follow our blocbook project ports [table here](https://github.com/trezor/blockbook/blob/51d10bccd207e61eb0ed32702635b6f3f8ea33de/docs/ports.md) Let. me know if it's ok or we should change it.

Reply:
Ah okay then :shrug:. If it is documented there let's keep it.